apps/setreg

```diff
@@ -26,179 +26,168 @@ else
-					#  "NEAST" 10
-					#  "NMAfrica" 11
-					#  "MEAST" 12
-					#  "CentralAsia" 13
-					#  "India" 14
-					#  "Israel" 15
-					#  "Taiwan" 16
-					#  "MSA 2 (Chile)" 17
-				sleep 1
-		echo -ne "--- Import media\n" | $TEE -a $LOG
-		on -f mmx $PC i:0:0x50186010 1
-		sleep 1
-		echo -ne "--- Media Ripping\n" | $TEE -a $LOG
-		on -f mmx $PC i:0:0x50186020 1
-		sleep 1
-		echo -ne "--- Regionalcodes DVD\n" | $TEE -a $LOG
-		on -f mmx $PC i:28442848:515 2
-		sleep 1
-		echo -ne "--- Regionalcodes Blueray\n" | $TEE -a $LOG
-		on -f mmx $PC i:28442848:516 2
-		sleep 1
-		echo -ne "--- R. Codes DVD Counter\n" | $TEE -a $LOG
-		on -f mmx $PC i:28442848:517 5
-		sleep 1
-		echo -ne "--- R. Codes Blueray Counter\n" | $TEE -a $LOG
-		on -f mmx $PC i:28442848:518 5
-		sleep 1
-		echo -ne "--- TMCPro EU\n" | $TEE -a $LOG
-		on -f mmx $PC i:28442848:520 32783
-		sleep 1
-		echo -ne "--- Gracenote local ON\n" | $TEE -a $LOG
-		on -f mmx $PC b:0:3221356628:1.6 1
-		on -f mmx $PC b:0:3221356628:1.7 1
-		sleep 1
-		echo -ne "--- Radio stationlogo DB\n" | $TEE -a $LOG
-		on -f mmx $PC b:0:3221356628:6 1
-			#	"none" 0
-			#	"EU" 1
-			#	"Taiwan" EA
-			#	"Korea" CD
-			#	"Hongkong" 66
-			#	"China" 28
-			#	"Macao" 67
-			sleep 1
-		echo -ne "--- SDS region Flag\n" | $TEE -a $LOG
-		on -f mmx $PC b:0:3221356628:9 1
-			#	"no region" 0
-			#	"EU" 1
-			#	"NAR" 2
-			#	"MSA" 3
-			#	"Korea" 4
-			#	"China" 5
-			#	"Japan" 6
-			#	"AsiaPacific" 7
-			#	"Australia" 8
-			#	"South Africa" 9
-			#	"NEAST" 10
-			#	"NMAfrica" 11
-			#	"MEAST" 12
-			#	"CentralAsia" 13
-			#	"India" 14
-			#	"Israel" 15
-			#	"Taiwan" 16
-			#	"MSA 2 (Chile)" 17
-			#	"China 2" 18
-			#	"China 3" 19
-			sleep 1
-		echo -ne "--- variant region\n" | $TEE -a $LOG
-		on -f mmx $PC i:0x286f058c:9 2
-			#   "Europe/ROW" 1
-			#   "Europe" 2  
-			#   "NAR" 3
-			#   "ROW" 4 
-			#   "China" 5
-			#   "Japan" 6   
-			#   "Korea" 7
-			#   "Asia" 8
-			#   "Taiwan" 9
-			sleep 1
-		#echo -ne "switch variant feature byte type to High\n"
-		#on -f mmx $PC i:0x286f058c:4 1
-		#	sleep 1
-		echo -ne "--- disable variant feature byte sirius\n"
-		on -f mmx $PC i:0x286f058c:8 0
-			sleep 1
-		echo -ne "--- store changes to persistence\n\n"
-		on -f mmx $PC b:0:1 0
-			sleep 2
-		[ -z "$noboot" ] && echo -ne "\nAll done!\n"
-	else
-		echo -ne "no EU train found - will stop here\n" | $TEE -a $LOG
-		exit 1
-	fi
-
-# return 2> /dev/null
-
-trap 2
-
-[ -z "$noboot" ] && . $thisdir/reboot -t 10
-
+		trap '' 2
+		mount -uw /net/rcc/mnt/efs-persist
+		if [[ "$TRAINVERSION" = *ER* ]]; then
+			echo -ne "Writing EU language datasets to 5F...\n" | $TEE -a $LOG
+			/net/mmx/fs/sda0/apps/setlang -eu -noboot
+			echo -ne "\nChanging coding and adaptations to EU:" | $TEE -a $LOG
+			echo -ne "\n--- Summertime algo.\n" | $TEE -a $LOG
+			# 0=none, 1=manual, 2=MESZ, 3=USA
+			if [ -f /net/mmx/mnt/app/eso/bin/apps/pc ]; then
+				on -f mmx $PC i:28442848:503 2
+			else
+				$PERSW -t int 28442848 503 2
```

Comment:
persw `-f` missing. We have to test if each or just last command needs this.
Again we should remove `pc?